packages/bappo-components/src/components/Avatar/types.ts

```diff
@@ -0,0 +1,9 @@
+export type AvatarProps = {
+  backgroundColor: string;
```

Comment:
if backgroundColor is must define, do we still need default value for that

Reply:
I've removed the default value for the required backgroundColor